feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/domain/validators/ValidatorProvider.kt

```diff
@@ -52,14 +52,16 @@ class ValidatorProvider(
 
         return requestedValidatorIds.map { accountIdHex ->
             val prefs = validatorPrefs[accountIdHex]
+            val maxNominators = stakingConstantsRepository.maxRewardedNominatorPerValidator()
```

Comment:
Can be fetched outside of the map to avoid being fetched` len(validators)` times

Reply:
Done